### DISCUSSION TITLE
Don't think I am getting chia from farming

### DISCUSSION THREAD

**Discussion by jasonhara** [Chia-Network/chia-blockchain#17327] [Support]
I have the same amount of chia for months

**Comment by BrandtH22** [comment]
@jasonhara , please note that I have removed the original attachment. For security reasons we are not able to download and open .docx files, can you please provide screenshots? 

The first thing to check is generally that the full node is synced and that it is correctly providing quality proofs while keeping in mind that going 6x the ETW is well within margin for luck.

We can generally provide more timely and thorough support in our discord server (https://discord.gg/chia)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
